src/cleanfigure.m

```diff
@@ -61,14 +64,17 @@ function cleanfigure(varargin)
   m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'targetResolution', 600, @isValidTargetResolution);
 
   m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'minimumPointsDistance', 1.0e-10, @isnumeric);
+  m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'scalePrecision', 1, @isnumeric);
```

Comment:
Now that I think of it, we probably should also define a value where the whole function is disabled. E.g. `scalePrecision <= 0` or `NaN` could be reasonable things to do.